src/main/java/org/traccar/protocol/RuptelaProtocolDecoder.java

```diff
@@ -151,9 +176,19 @@ private void decodeParameter(Position position, int id, ByteBuf buf, int length)
                     position.set(Position.KEY_ALARM, Position.ALARM_ACCIDENT);
                 }
                 break;
+            case 415:
+                if (readValue(buf, length, false) != 1) {
+                    position.set(Position.KEY_ALARM, Position.ALARM_GPS_ANTENNA_CUT);
+                }
+                break;
             case 645:
                 position.set(Position.KEY_OBD_ODOMETER, readValue(buf, length, true) * 1000);
                 break;
+            case 758:
+                if (readValue(buf, length, false) == 1) {
```

Comment:
Can this also be `> 0`?

Reply:
This one not, 0 -> no tampering, 1 -> tampering activated and 255 - no data from tampering switch, which is present in my tests when there is no tampering

Reply:
I see. Sounds good.